ref/general/oidc.adoc

```diff
@@ -4,51 +4,44 @@
 :page-categories: OpenID Connect
 :seo-title: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0.
 :seo-description: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0 that is widely used in SSO technologies. OIDC works well with cloud, mobile, and native applications.
-= Configuring Open Liberty as a single sign-on provider
+= Configuring Open Liberty as a single sign-on provider.
 
-In an increasingly fragile world of the internet, user credentials are precious resources that need to be protected. With single sign-on (SSO), users can log in easily without worrying that their credentials are being disclosed. Open Liberty helps you with identity management.
+Single sign-on is a process to sign in once with your credentials to access multiple applications. OpenID Connect (OIDC) is a protocol that helps developers to authenticate users without the need to manage user credentials.
```

Comment:
Change
"helps developers to authenticate"
To 
"helps developers authenticate"